homeassistant/components/cover/mqtt.py

```diff
@@ -153,6 +212,11 @@ def current_cover_position(self):
         """
         return self._position
 
+    @property
+    def device_class(self):
```

Comment:
As described above, if the only function is user customization, we'll leave this to the customize config. This property can be dropped.